sdk/template/Azure.Template/README.md

```diff
@@ -93,6 +93,10 @@ If the package or a related package supports it, include tips for logging or ena
 * If appropriate, point users to other packages that might be useful.
 * If you think there's a good chance that developers might stumble across your package in error (because they're searching for specific functionality and mistakenly think the package provides that functionality), point them to the packages they might be looking for.
 
+## Contributing
+
+This is a template, but your SDK readme should include details on how to contribute code to the repo/package.
```

Comment:
Do we want every readme to contain how to contribute code? I'd expect that to be in the root readme.

Reply:
Did this because it's part of the `example` readme and this is a section that's present there. We can edit per repo section requirements as needed tho.